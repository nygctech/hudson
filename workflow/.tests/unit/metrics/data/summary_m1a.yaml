section_information:
  cycles: 5
  experiment: 20210323_4i4color
  machine: Origin
  planesizeMB: 8
  rawpath: /gpfs/commons/groups/innovation/sarah/metrics_test/20210323_4i4color/raw_zarr
  tiles: 1
