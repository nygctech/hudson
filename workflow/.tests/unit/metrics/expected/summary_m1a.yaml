section_information:
  cycles: 5
  experiment: 20210323_4i4color
  machine: Origin
  planesizeMB: 8
  rawpath: /gpfs/commons/groups/innovation/sarah/metrics_test/20210323_4i4color/raw_zarr
  tiles: 1
segmentation:
  avergae_cell_size: 23.6776
  cell_size_standard_deviation: 10.6845
  number_of_cells: 1958
